ports/samd/machine_wdt.c

```diff
@@ -24,9 +24,7 @@
  * THE SOFTWARE.
  */
 
-#include "py/runtime.h"
 #include "modmachine.h"
-
 #include "sam.h"
 
```

Comment:
Suggest adding a comment at the top of these files along the lines of "This file is never compiled standalone, it's included directly from extmod/machine_wdt.c" - as this connection is hard to see otherwise.

Reply:
Comment now added to all implementation files.